src/asciidoc/web-mvc.adoc

```diff
@@ -1257,15 +1257,20 @@ The following are the supported return types:
 * An `HttpHeaders` object to return a response with no body.
 * A `Callable<?>` can be returned when the application wants to produce the return value
   asynchronously in a thread managed by Spring MVC.
+* An `WebAsyncTask<?>` can be returned when the application wants to produce the return
+  value from a thread of its own choosing.
 * A `DeferredResult<?>` can be returned when the application wants to produce the return
   value from a thread of its own choosing.
 * A `ListenableFuture<?>` can be returned when the application wants to produce the return
   value from a thread of its own choosing.
+* A class which implements a `CompletionStage<?>` interface (e.g. `CompletableFuture<?>`)
```

Comment:
Change to:

A class which implements `CompletionStage<?>` (e.g., `CompletableFuture<?>`)